commands/moderation/purge.js

```diff
@@ -38,7 +37,7 @@ module.exports = class PurgeCommand extends Command {
         message
           .reply(`🗑️ Sucessfully Deleted ${args.purgecount} messages.`)
           .then(async e => {
-            await e.delete(2000);
+            await e.delete({ timeout: 2000 });
```

Comment:
in discord.js v12 u need to put the time to an object. Refer [this](https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=delete)